### ISSUE TITLE
Upload psd files

### ISSUE THREAD

**Issue by Daweo93** [avocode/avocode#1350]
When I tried to upload PSD file.s

-----
App v. `2.13.1`
Avocode UID: 81652
```
TypeError: Cannot read property 'offset' of undefined
    at CellSizeAndPositionManager.getVisibleCellRange (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/utils/CellSizeAndPositionManager.js:223:21)
    at ScalingCellSizeAndPositionManager.getVisibleCellRange (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/utils/ScalingCellSizeAndPositionManager.js:154:47)
    at Grid._calculateChildrenToRender (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/Grid.js:616:65)
    at Grid.componentWillUpdate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/Grid.js:503:12)
    at ReactCompositeComponentWrapper._performComponentUpdate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:714:14)
    at ReactCompositeComponentWrapper.updateComponent (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:645:12)
    at ReactCompositeComponentWrapper.performUpdateIfNecessary (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:561:12)
    at Object.performUpdateIfNecessary (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactReconciler.js:157:22)
    at runBatchedUpdates (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:150:21)
    at ReactReconcileTransaction.perform (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/Transaction.js:140:20)
    at ReactUpdatesFlushTransaction.perform (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/Transaction.js:140:20)
    at ReactUpdatesFlushTransaction.perform (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:89:32)
    at flushBatchedUpdates (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:172:19)
    at ReactUpdatesFlushTransaction.close (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:47:7)
    at ReactUpdatesFlushTransaction.closeAll (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/Transaction.js:206:25)
    at ReactUpdatesFlushTransaction.perform (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/Transaction.js:153:16)
    at ReactUpdatesFlushTransaction.perform (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:89:32)
    at Object.flushBatchedUpdates (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:172:19)
    at ReactDefaultBatchingStrategyTransaction.closeAll (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/Transaction.js:206:25)
    at ReactDefaultBatchingStrategyTransaction.perform (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/Transaction.js:153:16)
    at Object.batchedUpdates (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactDefaultBatchingStrategy.js:62:26)
    at Object.enqueueUpdate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:200:22)
    at enqueueUpdate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactUpdateQueue.js:24:16)
    at Object.enqueueSetState (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactUpdateQueue.js:209:5)
    at t.ReactComponent.setState (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactComponent.js:63:16)
    at t.r._handleUserDataNotificationStateChange (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:20:10670)
    at emitNone (events.js:91:20)
    at t.emit (events.js:185:7)
    at t.emitChange (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/flux/lib/store.js:47:12)
    at n._handleReceivePageAction (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:28:26940)
    at Object.ID_11 (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/flux/lib/utils.js:11:14)
    at Dispatcher._invokeCallback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/flux/lib/Dispatcher.js:198:24)
    at Dispatcher.dispatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/flux/lib/Dispatcher.js:174:14)
    at Dispatcher.dispatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/flux/lib/dispatcher.js:76:28)
    at t.dispatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/flux/lib/action-creator.js:37:24)
    at t.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:18:4530)
    at e.<anonymous> (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:18:9526)
    at tryCatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/regenerator-runtime/runtime.js:64:40)
    at Generator.invoke [as _invoke] (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/regenerator-runtime/runtime.js:355:22)
    at Generator.prototype.(anonymous function) [as next] (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/regenerator-runtime/runtime.js:116:21)
    at r (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:18:7794)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:18:7894
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

**Comment by petrbrzek** [comment]
Thanks for reporting. We know about this bug and the fix should be released tomorrow.

**Comment by jakubzitny** [comment]
This should be fixed in Avocode `2.13.2` we've just released. Let us know if this problem persists or if you have any other issues with it.

Thanks.
